### ISSUE TITLE
Call to undefined method when using kernel browser

### ISSUE THREAD

**Issue by nusje2000** [symfony/panther#420]
When you use `assertPageTitleContains` when using a KernelBrowser, then the following exception is thrown:
```
1) App\Tests\AppBundle\Controller\RegistrationControllerTest::testRegisterPageLoads
Error: Call to undefined method Symfony\Bundle\FrameworkBundle\KernelBrowser::createWebDriverByFromLocator()

C:\project_dir\vendor\symfony\panther\src\WebTestAssertionsTrait.php:224
C:\project_dir\vendor\symfony\panther\src\WebTestAssertionsTrait.php:56
C:\project_dir\vendor\symfony\framework-bundle\Test\DomCrawlerAssertionsTrait.php:69
C:\project_dir\vendor\symfony\panther\src\WebTestAssertionsTrait.php:94
C:\project_dir\tests\AppBundle\Controller\RegistrationControllerTest.php:24
```

**Comment by nusje2000** [comment]
I'm working on a fix

**Comment by dunglas** [comment]
Closed by #423.
